### PR TITLE
develop <- Feature/70: services in react

### PR DESCRIPTION
**Pull request by blablanumerodeux**
…ce class

### CODE REVIEW

File: src/components/create-journey/CreateJourney.js [hunk 1]
```diff
@@ -14,45 +15,38 @@ class CreateJourney extends Component {
...
+     * @param {} e the onChange event 
      */
-    handleChange = function(e) {
-        // console.log(e.target)
-        switch(e.target.name){
-            case 'journey-id':
-                this.setState({ journeyId: e.target.value });
-                break;
+    handleChange = function (e) {
+        console.log(e.target.name);
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js]: pourquoi garder le console.log?

File: src/components/create-journey/CreateJourney.js [hunk 2]
```diff
@@ -14,45 +15,38 @@ class CreateJourney extends Component {
...
+        switch (e.target.name) {
             case 'destination':
                 this.setState({ destination: e.target.value });
                 break;
             case 'price':
                 this.setState({ price: e.target.value });
                 break;
             default:
-                console.error('no such state attribut')
+                console.error('No such state attribut')
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js]:
```suggestion
                console.error('No such state attribut');
```

File: src/components/create-journey/CreateJourney.js [hunk 3]
```diff
@@ -62,24 +56,25 @@ class CreateJourney extends Component {
         const variables = {};
 
         try {
-            axios.post('http://localhost:8080/graphql', {
+            // is there a way to use Relay here instead of directly axios ?
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js:59]: probably

File: src/components/create-journey/CreateJourney.js [hunk 4]
```diff
@@ -62,24 +56,25 @@ class CreateJourney extends Component {
...
+            axios.post('http://localhost:8080/graphql', {// TODO export this URL into a config file
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js:60]: yes

File: src/components/create-journey/CreateJourney.js [hunk 5]
```diff
@@ -62,24 +56,25 @@ class CreateJourney extends Component {
...
-                    journey: response.data.data.createJourney
+                    journey: response.data.data.createJourney // put the Journey that we received from the back in the state
                 }));
-                console.dir(this.state)
+                console.dir(this.state);
             });
         } catch (error) {
             // If there's an error, set the error to the state
-            console.log('error : ' + error);
+            console.log('error: ' + error);
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js:77]: I don't like it

File: src/components/create-journey/CreateJourney.js [hunk 6]
```diff
@@ -95,21 +90,20 @@ class CreateJourney extends Component {
...
                 <input id="destination" type="text" name="destination" onChange={this.handleChange} />
-                <br/>
-                <label htmlFor="price">Create a journey : </label>
+                <br />
+                <label htmlFor="price">Price: </label>
                 <input id="price" type="text" name="price" onChange={this.handleChange} />
-                <br/>
-                <button onClick={this.createJourney}>Validate</button>
+                <br />
+                <button onClick={this.createJourney}>Create</button>
```
Comment (shulolo) [inline src/components/create-journey/CreateJourney.js:106]: Validate ça m'allait mais as you want

File: src/components/journey-display/JourneyDisplay.js [hunk 7]
```diff
@@ -25,15 +25,15 @@ class JourneyDisplay extends Component {
     /**
      * Bring a journey from the server (graphql)
      */
-    getJourney = async (query, variables) => {
+    getJourney = async (query, variables) => { //TODO: see if we can use RxJs instead of async await
         try {
-            const response = await axios.post('http://localhost:8080/graphql', {
+            const response = await axios.post('http://localhost:8080/graphql', {// TODO: put the URL in a properties file
```
Comment (shulolo) [inline src/components/journey-display/JourneyDisplay.js]:
```suggestion
            const response = await axios.post('http://localhost:8080/graphql', { // TODO: put the URL in a properties file
```

**Review by shulolo — CHANGES_REQUESTED** [review]

File: src/components/create-journey/CreateJourney.js (hunk 1 above)
Comment (blablanumerodeux) [reply src/components/create-journey/CreateJourney.js]: I'll remove it 
```suggestion
        // console.log(e.target.name);
```

**Review by shulolo — APPROVED** [review]

**Pull request merged**